Packs/CloudIncidentResponse/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -1019,6 +1206,23 @@ inputs:
   required: false
   description: Whether to execute the user remediation flow automatically.
   playbookInputQuery:
+- key: credentialsRemediationType
+  value: {}
```

Comment:
if we dont have a default value, we should add if this input isnt empty for the conditional task whether to run the creds rotation playbook